frontend/src/actions/transfers/subsidized-transfer.ts

```diff
@@ -33,6 +33,22 @@ export class SubsidizedTransfer extends Transfer {
     this.sourceChain.requestManagerAddress = this.originalRequestManagerAddress;
   }
 
+  async withdraw(provider: IEthereumProvider): Promise<void> {
+    if (this.feeSubAddress) {
```

Comment:
Does this take into account that the fee sub contract is only called upon requests which were created by the feesub?

Reply:
Well, the fact that we are inside a `SubsidizedTransfer` instance is implying that we used a FeeSub contract for that same request.
Whether we should create a normal `Transfer` or a `SubsidizedTransfer` is decided here: https://github.com/beamer-bridge/beamer/blob/main/frontend/src/composables/useTransferRequest.ts#L57-L69

Reply:
ok makes sense 👍